domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -429,6 +429,24 @@ class TopicListController @Inject constructor(
     }
   }
 
+  /*
+  * # Dependency graph:
+  *
+  *      Fractions
+  *         |
+  *        |
+  *       v
```

Comment:
Should these be shifted to represent an arrow?

Reply:
I tried shifting it but it doesn't make perfect arrow.

Reply:
I think if the initial space is consistent it should work.

Reply:
@veena14cs PTAL

Reply:
I tried giving consistent spacing, but didn't workout.

Reply:
@BenHenning Any specific reason why it might not be getting aligned in line?

Reply:
@rt4914 I have updated code please check. Also there is difference in how it shows up in Github and in Android Studio. I am attaching screenshot for reference.
![Screenshot (13)](https://user-images.githubusercontent.com/16301028/108101920-ea732580-70ad-11eb-9aa2-080b54f44811.png)

Reply:
Looks correct. Thanks. It should look correct in android studio.

Reply:
For me it looks like this:
<img width="778" alt="Screenshot 2021-02-17 at 11 01 43 AM" src="https://user-images.githubusercontent.com/9396084/108160692-926d0b00-710f-11eb-9cec-1908ec410d29.png">

Reply:
My Android studio is weird :).